ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -0,0 +1,622 @@
+ /*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "include/core/SkFont.h"
+#include "include/core/SkFontMgr.h"
+
+#include "ReactSkia/utils/RnsUtils.h"
+
+#include "OnScreenKeyBoard.h"
+#include "OSKConfig.h"
+#include "OSKLayout.h"
+
+namespace OSK {
+
+    std::mutex oskLaunchExitCtrlMutex;//For synchronized OSK Launch & Exit
```

Comment:
is there any indentation issue here ?
should allthe functions be starting from beginning of line

Reply:
Intentionally shifting teh start of  Api's inside namespace, to give an understanding at quick look it is enclosed or part of " something".

Please let me know , if it not adhere to common standard or standard we have defined.

Reply:
As discussed, will address this comment.